provider/kubernetes/kubernetes_test.go

```diff
@@ -1718,15 +1759,23 @@ func TestIngressAnnotations(t *testing.T) {
 					},
 				},
 			},
+			"rewrite/api": {
+				Backend:        "rewrite/api",
+				PassHostHeader: true,
+				Routes: map[string]types.Route{
+					"/api": {
+						Rule: "ReplacePath:/",
+					},
+					"rewrite": {
+						Rule: "Host:rewrite",
+					},
+				},
+				Priority: len("/api"),
+			},
 		},
 	}
 
-	actualJSON, _ := json.Marshal(actual)
-	expectedJSON, _ := json.Marshal(expected)
-
-	if !reflect.DeepEqual(actual, expected) {
-		t.Fatalf("expected %+v, got %+v", string(expectedJSON), string(actualJSON))
-	}
+	assert.Equal(t, expected, actual, "The rewritten data did not match the expected.")
```

Comment:
Removed the extra message - legacy of JUnit where you don't always get a "nice" error message, and spitting out some words helped make more sense of things.